netci.groovy

```diff
@@ -184,4 +184,46 @@ Utilities.addGithubPRTrigger(prCCJob, 'Code Coverage Windows Debug', '@dotnet-bo
         Utilities.addStandardOptions(newJob)
         Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
     }
+
+// **************************
+// Outerloop and Innerloop against the latest dependencies on Windows. Rolling daily for debug and release
+// **************************
+
+['Debug', 'Release'].each { configuration ->
+    def configurationJobName = configuration.toLowerCase()
+    def jobName = "LatestDependencies_windows_${configurationJobName}"
+
+    // Create the new rolling job
+    def newLatestDepRollingJob = job(Utilities.getFullJobName(project, jobName, false)) {
+        label('windows-elevated')
+        steps {
+            batchFile("build.cmd /p:Configuration=Windows_NT_${configuration}  /p:FloatingTestRuntimeDependencies=true /p:WithCategories=\"InnerLoop;OuterLoop\"")
+        }
+    }
+
+    // Add commit job options
+    Utilities.addScm(newLatestDepRollingJob, project)
+    Utilities.addStandardNonPRParameters(newLatestDepRollingJob)
+    Utilities.addPeriodicTrigger(newLatestDepRollingJob, '@daily')
+
+    // Create the new PR job for on demand execution.  No automatic PR trigger.
+    // Triggered with '@dotnet-bot test latest dependencies please'
+
+    def newLatestDepPRJob = job(Utilities.getFullJobName(project, jobName, true)) {
+        label('windows-elevated')
+        steps {
+            batchFile("build.cmd /p:Configuration=Windows_NT_${configuration}  /p:FloatingTestRuntimeDependencies=true /p:WithCategories=\"InnerLoop;OuterLoop\"")
+        }
+    }
+
+    // Add a PR trigger
+    Utilities.addGithubPRTrigger(newLatestDepPRJob, "Outerloop and Innerloop against the latest dependencies Windows ${configuration} Build and Test", '@dotnet-bot test latest dependencies please')
```

Comment:
This is a super long name. Consider just "Latest package dependencies ${os} ${configuration} Build and Test"